backend/benches/build_map_model.rs

```diff
@@ -9,18 +9,14 @@ fn benchmark_build_map_model(c: &mut Criterion) {
     ] {
         // Do the file i/o (reading OSM.xml) outside of the bench loop
         let map_model_builder = neighbourhood.map_model_builder().unwrap();
-        c.bench_function(
-            &format!(
-                "build map_model: {name}",
-                name = neighbourhood.study_area_name
-            ),
-            |b| {
+        c.benchmark_group(neighbourhood.savefile_name)
+            .sample_size(neighbourhood.bench_sample_size())
```

Comment:
Some of the Stasbourg benches were painfully slow, so I've lowered the sample size for them.